woodwork/statistics_utils.py

```diff
@@ -306,3 +338,79 @@ def _get_value_counts(dataframe, ascending=False, top_n=10, dropna=False):
         values = list(df.to_dict(orient="index").values())
         val_counts[col] = values
     return val_counts
+
+
+def _get_top_values_numeric(series, _range):
+    """Get the most frequent values in a range of integers. Will ignore
+        null values.
+
+    Args:
+        series (pd.Series): data to use to find most frequent values
+        _range (type(range)): sequence of integers for which to count occurrences in series
+
+    Returns:
+        top_list (list(dict)): a list of dictionary with keys `count` and
+            `value`
+    """
+    frequencies = series.value_counts(dropna=True)
+    return [
+        {"count": frequencies[i] if i in frequencies else 0, "value": i} for i in _range
+    ]
+
+
+def _get_top_values_categorical(series, num_x):
+    """Get the most frequent values in a pandas Series. Will exclude null values.
+
+    Args:
+        column (pd.Series): data to use find most frequent values
+        num_x (int): the number of top values to retrieve
+
+    Returns:
+        top_list (list(dict)): a list of dictionary with keys `count` and
+            `value`
+    """
+    frequencies = series.value_counts(dropna=True)
+    df = frequencies.head(num_x).reset_index()
+    df.columns = ["value", "count"]
+    top_lt = list(df.to_dict(orient="index").values())
+    top_lt = sorted(top_lt, key=lambda i: (i["count"], i["value"]))
+    return top_lt
+
+
+def _get_recent_values(column, num_x):
+    """Get the most frequent, recent values for a given datetime column.
```

Comment:
Reading this, I'm not sure if we're getting the "n most recent dates and listing the counts for each" or the "n most frequent dates sorted by recency" or something. The recent values language in general is a bit confusing and could maybe be a bit more consistent.

Reply:
Updated the wording for this. The function was actually getting the value counts for the `n` oldest dates in the series. Updated the function name and the docstring.